ntbs-service-unit-tests/Services/NotificationServiceTest.cs

```diff
@@ -197,22 +197,6 @@ public async Task UkBorn_IsSetToCorrectValueDependentOnBirthCountry(Country coun
             VerifyUpdateDatabaseCalled();
         }
 
-        [Fact]
-        public async Task UkBorn_IsSetToNullIfCountryNotSet()
-        {
-            // Arrange
-            var notification = new Notification();
-            var patient = new PatientDetails() { UkBorn = true };
-
-            // Act
-            await _notificationService.UpdatePatientDetailsAsync(notification, patient);
-
-            // Assert
-            Assert.Null(patient.UkBorn);
-            _mockContext.Verify(context => context.SetValues(notification.PatientDetails, patient));
-            VerifyUpdateDatabaseCalled();
-        }

```

Comment:
It would be good to replace this with a test on the `PatientDetailsDisplay` - it should be a simple test to add.

Reply:
This test case doesn't really exist anymore - the change means that we never have a null value for UkBorn. Do you think we should have null? As UkBorn is barely used and not in the database anymore it seems like probably not

Reply:
Sorry, I didn't mean that there's a direct mapping between the tests. I mean that we now have some new logic defining what we mean by "UK born" (relating to the country ID) in the `PatientDetailsDisplay` and we should test that logic. (We probably just want 3 test cases: country ID is 235, something else, or null).

Reply:
There is a test above this which is testing it. I will move it into a new test class though, as it doesn't fit here